app/views/statistics/index.html.slim

```diff
@@ -1,5 +1,13 @@
 .page-header
-  h2 Statistics
+  h1 Statistics
+
+= render 'periods_bar'
 
 = content_for :sidebar
   = render 'home/settings'
+
+= content_for :google_charts_api
+  = javascript_include_tag 'https://www.google.com/jsapi'
+  = javascript_include_tag 'statistics'
```

Comment:
@yunixon don't include statistics js separately. It should be included in `application.js`